src/Swagger/Client/OperationConfig.php

```diff
@@ -6,138 +6,140 @@
+
     protected $bodyParameter;
-    
+
     protected $formParameters = [];
-    
+
     public function getOperation()
     {
         return $this->operation;
     }
-    
+
     public function setOperation(OperationReference $operation)
     {
         $this->operation = $operation;
         return $this;
     }
-    
+
     public function getScheme()
     {
         return $this->scheme;
     }
-    
+
     public function setScheme($scheme)
     {
         $this->scheme = $scheme;
         return $this;
     }
-    
+
     public function getMediaType()
     {
         return $this->mediaType;
     }
-    
+
     public function setMediaType($mediaType)
     {
         $this->mediaType = $mediaType;
         return $this;
     }
-    
+
     public function getPathParameters()
     {
         return $this->pathParameters;
     }
-    
+
     public function setPathParameters($pathParameters)
     {
         $this->pathParameters = $pathParameters;
         return $this;
     }
-    
+
     public function addPathParameter($pathParameter)
     {
         $this->pathParameters[] = $pathParameter;
         return $this;
     }
-    
+
     public function getQueryParameters()
     {
         return $this->queryParameters;
     }
-    
+
     public function setQueryParameters($queryParameters)
     {
         $this->queryParameters = $queryParameters;
         return $this;
     }
-    
+
     public function addQueryParameter($queryParameter)
     {
         $this->queryParameters[] = $queryParameter;
         return $this;
     }
-    
+
     public function getHeaderParameters()
     {
         return $this->headerParameters;
     }
-    
+
     public function setHeaderParameters($headerParameters)
     {
         $this->headerParameters = $headerParameters;
         return $this;
     }
-    
+
     public function addHeaderParameter($headerParameter)
     {
         $this->headerParameters[] = $headerParameter;
         return $this;
     }
-    
+
     public function getBodyParameter()
     {
         return $this->bodyParameter;
     }
-    
+
     public function setBodyParameter($bodyParameter)
     {
         $this->bodyParameter = $bodyParameter;
         return $this;
     }
-    
+
     public function getFormParameters()
     {
         return $this->formParameters;
     }
-    
+
     public function setFormParameters($formParameters)
     {
         $this->formParameters = $formParameters;
         return $this;
     }
-    
+
     public function addFormParameter($formParameter)
     {
         $this->formParameters[] = $formParameter;
         return $this;
     }
-    
-    public function setFileUpload($filename, $formname, $data = null, $ctype = null)
+
+    public function setFileUpload($pathToFile, $name = null, $data = null, $ctype = null)
     {
-        $this->formParameters['formname'] = $formname;
-        $this->formParameters['filename'] = $filename;
+        $this->formParameters['pathToFile'] = $pathToFile;
```

Comment:
No, it was'nt change of our api.....but i tested it a lot (old our version) and found that test passed just because i accidentally used name 'file' for uploaded file and it was sent as formname.
If you'll use any other name - request was invalid. So i hardcoded formname and added particular parameter name to use as file name. Renamed first parameter to pathToFile for better readability and be sure that users will put there path to file, not simple name of file.